### PR TITLE
Move all Lambdas into a single project. This reduces code duplication.

### PR DESCRIPTION
**Pull request by jashelb90**
Combine current event processor and obfuscation functions into a single project.

What do we think of this approach?

If we like it we can bring in the Cleanse function as well and follow this pattern going forward.

### CODE REVIEW

File: event-processing/event-processing-template.yml [hunk 1]
```diff
@@ -229,9 +229,9 @@ Resources:
     Type: AWS::Serverless::Function
     Properties:
       FunctionName: "EventProcessorFunction-KBV"
-      CodeUri: ../event-processing/event-processor
+      CodeUri: ../event-processing/lambda
       PackageType: Zip
-      Handler: app.handler
+      Handler: app.eventProcessorHandler
```
Comment (des-hippo) [inline event-processing/event-processing-template.yml:234]: Small point, but I would rename the classes and keep the method name the same. Eg.
```
eventProcessorApp.handler
obfuscatorApp.handler 
```
etc

Comment (jashelb90) [reply event-processing/event-processing-template.yml:234]: So at the moment they are all in the same file. But yes I can look into having a separate class per handler if it is preferred

Comment (jashelb90) [reply event-processing/event-processing-template.yml]: @des-hippo Take a look at the changes I've made for this.

**Review by Aidan23Smith — APPROVED** [review]
Looks good to me

**Pull request merged**